OpenRA.Mods.Common/Traits/Air/AttackAircraft.cs

```diff
@@ -33,6 +33,9 @@ public class AttackAircraftInfo : AttackFollowInfo, Requires<AircraftInfo>
 		[Desc("Does this actor cancel its attack activity when it needs to resupply? Setting this to 'false' will make the actor resume attack after reloading.")]
 		public readonly bool AbortOnResupply = true;
 
+		[Desc("The angle relative to the actor's orientation used to fire the weapon from.")]
+		public readonly WAngle FiringAngle = WAngle.Zero;
```

Comment:
Apparently I was wrong about the convention here, so this should say `Facing` instead of `Angle`. Sorry 😓 
```suggestion
		public readonly WAngle[] FacingOffsets = { WAngle.Zero };
```
The suggestion also changes this to allow for multiple angles. (See the next comment.)

Reply:
fixed